README.md

```diff
@@ -40,12 +40,14 @@ python setup.py install
 
 ## Using
 
-Full documentation along with specific examples and tutorials are provided
-here: [https://toqito.readthedocs.io/](https://toqito.readthedocs.io/).
+Full documentation along with specific examples and tutorials are provided here:
+[https://toqito.readthedocs.io/](https://toqito.readthedocs.io/). 
 
 More information can also be found on the following
 [toqito homepage](https://vprusso.github.io/toqito/).
 
+Chat with us on [Discord](http://discord.unitary.fund/).
```

Comment:
```suggestion
Chat with us in our `Toqito` channel on [Discord](http://discord.unitary.fund/). 
```
I think I found an invite link here. I don't know if this will ever expire though. https://discordbotlist.com/servers/unitary-fund-764231928676089909

maybe using the link to the discord is better.